R/exclude.R

```diff
@@ -248,22 +249,20 @@ normalize_exclusions <- function(x, normalize_path = TRUE,
   x <- as.list(x)
   unnamed <- !nzchar(names2(x))
   if (any(unnamed)) {
-    # must be character vectors of length 1
     bad <- vapply(
       seq_along(x),
       function(i) {
         unnamed[i] && (!is.character(x[[i]]) || length(x[[i]]) != 1L)
       },
       logical(1L)
     )
+    bad_idx <- which(bad) # nolint: object_usage_linter.
```

Comment:
why define this instead of doing `which(bad)` inside the glue expression below? If we're going to define it, at least `any(bad)` should become `length(bad_idx) > 0L`.